src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -24,6 +25,10 @@ public void actionPerformed(ActionEvent e) {
             FiguresJoe.figureScript2(driverManager.getCurrentDriver());
         } else if (e.getActionCommand().equals("Figure Jane 1")) {
             FiguresJane.figureScript(new DriverAdapter(driverManager.getCurrentDriver()));
+        } else if (e.getActionCommand().equals("Square")) {
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.